docs/api-reference/advanced/all.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: All
+---
+
+## Basic Usage
+
+`all` indiscriminately matches every document in the index, assigning a uniform score of `1.0` to each.
```

Comment:
Should we document a use case for this here? At first read I'm not sure why would someone want to use this query.